src/appservice/MjolnirManager.ts

```diff
@@ -222,4 +246,56 @@ export class ManagedMjolnir {
     public get managementRoomId(): string {
         return this.mjolnir.managementRoomId;
     }
+
+    /**
+     * Intended to be called by the MjolnirManager to make sure the mjolnir is ready to listen to events.
+     * It should not be informed of any events via `onEvent` until this is called.
```

Comment:
What is `it`?
Shouldn't we throw an error (or capture an event with Sentry) if we're informed via `onEvent` before `start()` is called?